### PR TITLE
fancylogger: support replace rootlogger (and make it default in simple_option)

### PR DESCRIPTION
**Pull request by stdweird**

### CODE REVIEW

**Comment by stdweird** [comment]
still need to figure local fancylogger failing issue due to test ordering

File: lib/vsc/utils/fancylogger.py [hunk 1]
```diff
@@ -482,9 +493,9 @@ def _getCallingFunctionName():
         try:
             return inspect.stack()[2][3]
         except Exception:
-            return "?"
+            return "unkonwn__getCallingFunctionName"
```
Comment (JensTimmerman) [inline lib/vsc/utils/fancylogger.py]: fix typo

File: lib/vsc/utils/fancylogger.py [hunk 2]
```diff
@@ -510,9 +521,9 @@ def getRootLoggerName():
         try:
             return inspect.stack()[-1][1].split('/')[-1].split('.')[0]
         except Exception:
-            return "?"
+            return "unkown_getRootLoggerName"
```
Comment (JensTimmerman) [inline lib/vsc/utils/fancylogger.py]: fix typo

**Comment by stdweird** [comment]
@JensTimmerman fixed

**Comment by JensTimmerman** [comment]
bump version

**Pull request merged**